src/pages/settings/Profile/PersonalDetails/AddressPage.js

```diff
@@ -25,7 +25,7 @@ import FullscreenLoadingIndicator from '../../../../components/FullscreenLoading
 
 const propTypes = {
     /* Onyx Props */
-
+    country: PropTypes.string,
```

Comment:
This also needs docs description!
```suggestion
    country: PropTypes.string,

```